networkx/generators/joint_degree_seq.py

```diff
@@ -0,0 +1,283 @@
+"""Generate graphs with a given joint degree 
+"""
+#    BSD license.
```

Comment:
So, for example, you could make the top of the file look like:
```
#    Copyright (C) 2016 by
#    Minas Gjoka
#    All rights reserved.
#    BSD license.
#
# Author:  Minas Gjoka (minas.gjoka@gmail.com)
"""Generate graphs with a given joint degree """
import random
```

There is a description of file-header format in the NetworkX Github Wiki.
The decision on copyright and license is yours as described by @hagberg above.